packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js

```diff
@@ -40,8 +59,26 @@ export default class UserAccountSupportLevelCtrl {
   }
 
   areAllLevelsAvailable() {
-    return FULL_SUPPORT_LEVEL_SUBSIDIARIES.includes(
-      this.currentUser.ovhSubsidiary,
+    return !some(
```

Comment:
move this to SupportLevel class?

Reply:
I thought of this but it seems strange to pass `supportLevel` parameter or to handle an array of support levels in a model that is supposed to represent one